### ISSUE TITLE
Approvers want to alter requests based on the type of travel approved 

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/approvals#586]
Feedback from an approver:

```When I am reviewing requests that come to me, sometimes I need to go in and adjust the budget amounts. Use case examples: a) The requester has included meal expenses, but I am approving the travel as Professional Development travel (which doesn't cover meals); b) A case where the actual expense might be one amount, but I am only approving a certain amount as PART of the cost we are willing to cover; In both these kinds of cases, if I hit "request changes" it doesn't provide a response screen where I can say which changes I am requesting. In fact, my past practice was to simplify the process and just go in myself and edit the expenses directly (in the old system). The new system doesn't allow that, but it should. ```

**Comment by carolyncole** [comment]
I'm not really sure this is a good idea.  If the department head makes the changes how will we guarantee the person taking the trip will understand they are not getting money for food, ect.

**Comment by sdellis** [comment]
I agree with @carolyncole . We did take this into consideration and decided it would be best if the approver requests specific changes (there is in fact a "notes" field for this). 

We did discuss putting something prominent on the show screen and in the email that explains what is covered and what isn't based on the type of travel assigned, but I don't think we have a ticket for that yet. I think that would be a good idea since the travel policy is not easy to parse the way it's written.
